phoenicis-javafx/src/main/java/org/phoenicis/javafx/views/mainwindow/ui/LeftButton.java

```diff
@@ -29,7 +29,7 @@ public LeftButton(String name) {
         super(name);
         this.name = name;
         this.getStyleClass().add("leftButton");
-        this.setPrefWidth(Double.MAX_VALUE);
+        this.setMaxWidth(Double.MAX_VALUE);
```

Comment:
I believe you can set it to `Infinity` (see [here](https://stackoverflow.com/questions/26940341/how-to-set-fx-max-width-to-use-pref-size-in-javafx-css))